cmd/saturn-l2/main.go

```diff
@@ -288,6 +294,36 @@ func main() {
 	}
 }
 
+func logL1Connectivity(ctx context.Context, nConnectedL1s *atomic.Uint64) {
+	ticker := time.NewTicker(waitL1Connectivity)
+	defer ticker.Stop()
+
+	isConnected := false
+	isFirstEvent := true
+
+	for {
+		select {
+		case <-ticker.C:
+			if nConnectedL1s.Load() == 0 {
+				if isConnected || isFirstEvent {
+					fmt.Print("ERROR: Saturn Node is not able to connect to the network\n")
+					isConnected = false
+					isFirstEvent = false
+				}
+			} else {
+				if !isConnected || isFirstEvent {
+					fmt.Print("Saturn Node is connected to the network\n")
```

Comment:
In #62, I am adding the following message that's printed whenever the number of connected L1s changes to a non-zero value:

```go
fmt.Printf("INFO: Saturn Node is online and connected to %v peer(s)\n", nConnectedl1s)
```

When the number of connected L1s goes from 1 to 0, then I print the following message:

```go
fmt.Print("ERROR: Saturn Node lost connection to the network\n")
```

Could you please rework your PR to implement this behaviour?